docs/source/release/index.rst

```diff
@@ -12,6 +12,7 @@ Release Notes
    These release notes are for versions of ibis **1.0 and later**. Release
    notes for pre-1.0 versions of ibis can be found at :doc:`release-pre-1.0`
 
+* :support:`2977` Remove handwritten type parser; parsing errors that were previously `IbisTypeError` are now `parsy.ParseError`.
```

Comment:
also say that `parsy` is not a requirement

Reply:
IIRC these are listed in setup.py as well?

Reply:
`parsy` is a requirement though, is that what you meant?